synapse/tools/cryo/cat.py

```diff
@@ -32,30 +33,19 @@ def main(argv, outp=s_output.stdout):
         logger.setLevel(logging.INFO)
 
     if opts.ingest and not opts.jsonl and not opts.msgpack:
-        logger.error('Must specify exactly one of --jsonl or --msgpack if --ingest is specified')
+        outp.printf('Must specify exactly one of --jsonl or --msgpack if --ingest is specified')
         return 1
 
-    logger.info(f'connecting to: {opts.cryotank}')
+    outp.printf(f'connecting to: {opts.cryotank}')
```

Comment:
this *cannot* be a call to outp.printf() since we'll end up sending the string to stdout instead of stderr, preventing someone from redirecting the data to a unix pipe cleanly.

Reply:
great catch!  i'll fix. :)